### ISSUE TITLE
TON Journal

### ISSUE THREAD

**Issue by kanzaky** [ton-community/ton-docs#81]
### Tutorial title

TON JL

### Tutorial description

# TON JL -  Where activism is free
It would be interesting if we could have a tutorial on how to build an online journal using ton network, the news would be like an NFT I guess, and the users comment would be something in the same direction. I think this would be important for lots of journalist that would like to share critical and sensitive information reducing the chances of being target. We talking about freedom of speech and giving people the tools to share information and stay safe since we know the government and other entities don't like activists.

### Tutorial tags

#tonjl

### Skill level

Intermediate

### Hosted on ton.org/dev, blog.ton.org, blog.ton.org or hosted elsewhere?

Hosted on ton.org/dev

### For tutorials to be hosted on ton.org: Tutorial Content

The content cam be news related to people that shared information and pay the price for not being anonymous, like Snowden and WikiLeaks

**Comment by LevZed** [comment]
it sounds interesting, but it will turn out to be a full-fledged service, and not a tutorial in the documentation.

**Comment by SwiftAdviser** [comment]
Good idea for the tutorial! Do you want to make it? If yes, you could write a ton-footstep ([example](https://github.com/ton-society/ton-footsteps/issues/87)), provide a plan, and get compensation in USDT after the tutorial is finished.
